.github/workflows/ci.yml

```diff
@@ -22,3 +24,37 @@ jobs:
         run: |
           git-secrets --register-aws
           git-secrets --scan
+  doxygen:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Clone repository
+        uses: actions/checkout@v2
+        with:
+          submodules: recursive
+          path: freertos
+      - name: Install Python3
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.7.10'
```

Comment:
If you use a python-version of "3.7" or "3.8" github will automatically use the latest released version.

Reply:
Sure, changed